core/src/org/javarosa/form/api/FormEntryModel.java

```diff
@@ -397,7 +397,7 @@ public boolean isIndexRelevant(FormIndex index) {
             relevant = form.isRepeatRelevant(ref);
         } else {
             TreeElement node = form.getMainInstance().resolveReference(ref);
-            relevant = node.isRelevant(); // check instance flag first
+            relevant = (node != null) && node.isRelevant();
```

Comment:
Something is wrong with how heterogeneous paths are handled when checking relevancy (for <trigger> tags) that needs to be fixed. This just makes sure that if no node is found, we don't try to use it.